docs/sources/flow/reference/stdlib/regex_quote.md

```diff
@@ -0,0 +1,17 @@
+---
+aliases:
+- ../../configuration-language/standard-library/regex_quote/
+title: regex_quote
+---
+
+# regex_quote
+
+The `regex_quote` function returns a string that escapes all regular expression metacharacters inside the argument 
+text. The returned string is a regular expression matching the literal text.
+
+## Examples
+
+```
+> regex_escape("ip-10-0-69-8.ec2.internal")
```

Comment:
regex_escape or regex_quote ?

do we actually need/use this ? @rfratto can you share your feedback on this one ?

Reply:
no, it is not needed anymore.  Initially I was filtering nodes by regex as discovery relabel, but it is a wrong approach and discovery.kubernetes.selectors is now used.
I will remove it.

Reply:
removed